src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationTestFixture.cs

```diff
@@ -91,6 +80,14 @@ public HttpClient CreateClient()
             return client;
         }
 
+        /// <summary>
+        /// Creates a new instance of an <see cref="HttpClient"/> that can be used to
+        /// send <see cref="HttpRequestMessage"/> to the server.
+        /// </summary>
+        /// <param name="baseAddress">The base address of the <see cref="HttpClient"/> instance.</param>
+        /// <param name="handlers">A list of <see cref="DelegatingHandler"/> instances to setup on the
+        /// <see cref="HttpClient"/>.</param>
+        /// <returns>The <see cref="HttpClient"/>.</returns>
         public HttpClient CreateClient(Uri baseAddress, params DelegatingHandler[] handlers)
```

Comment:
Sure. One can argue that the user is going out of their way to break their own tests, but I'll add a check